apps/bestofjs-nextjs/src/server/api-hall-of-fame.tsx

```diff
@@ -0,0 +1,44 @@
+import { Data, FETCH_ALL_PROJECTS_URL } from "./api-utils";
+
+export function createHallOfFameAPI({
+  getData,
+}: {
+  getData: () => Promise<Data>;
+}) {
+  return {
+    async findMembers() {
+      const { populate, projectsBySlug } = await getData();
```

Comment:
`getData` can be undefined here right?

Reply:
`getData` is part of the context that each of the 3 APIs receive, check the types in VSCode:

![image](https://github.com/bestofjs/bestofjs/assets/5546996/21a2c41a-a2f0-46c2-b041-9ae521f17ea4)

Maybe I should make it more explicit by creating a specific type `Context`.

Reply:
Done here: https://github.com/bestofjs/bestofjs/pull/200/commits/8d7b57b050efd060720d3ead67f00d5e72464d95